willa/tind/format_validate_pymarc.py

```diff
@@ -0,0 +1,90 @@
+"""Fetches required fields from a pymarc record. 
+Sets value to None in cases where field is not present in marc record"""
```

Comment:
The top-level module docstring is a summary of methods in a module.  We don't need to duplicate the documentation of individual methods.

```suggestion
"""
Formats and validates PyMARC records for Willa.
"""
```